test/test_data.rb

```diff
@@ -145,7 +145,8 @@ def test_package
          [{:type=>"ssn_trace",
            :subtype=>nil},
           {:type=>"county_criminal_search",
-           :subtype=>"7years"}]}
+           :subtype=>"7years"}],
+       :requires_invitation => false}
```

Comment:
is there a reason this can't be formatted?
```rb
{
  :id=>"e44aa283528e6fde7d542194",
  :object=>"package",
  :uri=>"/v1/packages/e44aa283528e6fde7d542194",
  :created_at=>"2018-01-18T12:34:00Z",
  :name=>"Criminal Pro",
  :slug=>"criminal_pro",
  :price=>"6500",
  :screenings=>[
    {
      :type=>"ssn_trace",
      :subtype=>nil
    },
    {
      :type=>"county_criminal_search",
      :subtype=>"7years"
    }
  ],
  :requires_invitation => false
}
```

Reply:
@erdey No reason, I just kept the original formatting that the gem has. If I change it I should change all items in the test_data file, should i do it?

Reply:
I think it's reasonable if it will take < 10 minutes. Maybe a separate PR?